database/seeder.js

```diff
@@ -0,0 +1,22 @@
+const database = require('./connection.js');
+const random = require('./random.js');
+
+let iterator = 0;
+while (iterator < 16) {
+  const newSongLength = random.lengthInSecondsGenerator();
+  const newSongOptions = {
+    id: random.idGenerator(iterator),
+    artist: random.artistGenerator(),
+    title: random.titleGenerator(),
+    artwork: random.artworkGenerator(iterator), // ← This is a URL String
+    postingDate: random.postingGenerator(),
+    tag: random.tagGenerator(),
+    waveform: random.waveformGenerator(), // ← This is a URL String
+    lengthInSeconds: newSongLength,
+    currentTimestampInSeconds: random.currentTimestampGenerator(),
+    comments: random.commentsGenerator(newSongLength),
```

Comment:
Are you considering using the songNames in titles/lengths in the shared document?  It might cause problems if we actually try to sync up our components?  Although...perhaps this is beyond the scope of what we can feasibly accomplish before deployment.

Reply:
Yeah... for right now everything is pretty randomized. Depending on how far along we get I am more than happy to re-arrange my database documents to reflect "actual" data. It is a little out of scope for now, though.